### PR TITLE
Multicloud operations takeover

### PR DESCRIPTION
**Pull request by anthonydillon**
Blocked on https://github.com/canonical-websites/www.ubuntu.com/pull/3996

## Done
Build the multicloud operations takeover and remove the AI webinar takeover

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Check that homepage takeover matches [design](https://github.com/ubuntudesign/www.ubuntu.com-design/issues/153#issuecomment-418085878)

## Issue / Card
Fixes https://github.com/canonical-websites/www.ubuntu.com/issues/3957

### CODE REVIEW

**Comment by anasereijo** [comment]
@anthonydillon just realised the background is white. should be light grey

**Comment by anthonydillon** [comment]
@anasereijo made the title thin font and the images left and right background images are light grey so you don't see them if you set the background to light grey.

File: templates/takeovers/_multicloud-operations.html [hunk 1]
```diff
@@ -0,0 +1,77 @@
+<section class="p-strip--image is-deep js-takeover is-bordered p-takeover--multicloud-operations">
```
Comment (caldav) [inline templates/takeovers/_multicloud-operations.html]: Please add as another class: ` {% if not nojs_fallback %}u-hide{% endif %}` 

(see `takeovers/_tackling_iot.html` for example)

File: templates/takeovers/_multicloud-operations.html [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  <div class="row u-vertically-center">
+    <div class="col-7" itemscope="" itemtype="https://schema.org/Event">
+      <p class="p-muted-heading" style="font-size: 1rem;">CIO Guide to multi-cloud operations</p>
+      <h1 class="u-no-padding--top" style="font-weight: 100"><span class="event-title" itemprop="name">Choosing a cost effective cloud architecture</span></h1>
+      <p class="u-hide--small u-show--medium u-show--large"><a itemprop="url" href="/engage/multi-cloud-guide?utm_source=takeover&utm_campaign=FY19_Cloud_Openstack_Whitepaper_Multicloud" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Homepage Link', 'eventAction' : 'Choosing a cost effective cloud architecture takeover', 'eventLabel' : 'Download whitepaper', 'eventValue' : undefined });" class="p-button--positive">Download whitepaper</a></p>
```
Comment (caldav) [inline templates/takeovers/_multicloud-operations.html]: Please remove the GA event, they are auto-generated for all links.

File: templates/takeovers/_multicloud-operations.html [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+                  <path d="M101.1,34.2 C104.3,34.6 106.7,37.6 106.7,40.9 C106.7,41.1 106.7,41.3 106.7,41.5 C110.1,41.7 112.9,44.5 112.9,48 C112.9,51.6 109.9,54.5 106.3,54.5 L89.6,54.5 C86.4,54.5 83.7,52.2 83.1,49.2" id="Stroke-5_1_" stroke="#333333" stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round"></path>
+                </g>
+              </g>
+            </g>
+          </g>
+        </g>
+        <image src="https://assets.ubuntu.com/v1/3899d03f-choosingacloud.svg" xlink:href="">
+        </svg>
+      </div>
+      <p class="u-hide--medium u-hide--large u-show--small u-align--center"><a itemprop="url" href="https://pages.ubuntu.com/ubuntu_product_month.html" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Homepage Link', 'eventAction' : 'Ubuntu Product Month takeover', 'eventLabel' : 'Find out more', 'eventValue' : undefined });" class="p-button--positive">Find out more</a></p>
```
Comment (caldav) [inline templates/takeovers/_multicloud-operations.html]: This is an old CTA, should be the same as above

**Comment by anasereijo** [comment]
@anthonydillon sorry I just checked the svg images and they have multiply blending option selected. I resaved them as png - this way you should be able to have the light grey background

https://assets.ubuntu.com/v1/fca02000-suru-right-side.png
https://assets.ubuntu.com/v1/e63fb9cd-suru-left-side.png

**Comment by caldav** [comment]
@anthonydillon looks like you removed the UTMs but kept the GA events, that should be the other way around: links with UTMs, but no `onclick` GA event

**Comment by deadlight** [comment]
Small screen:
- No enough vertical spacing between illustration and the button
- Image overflows the container - give the svg max-width: 100% to fix.

image

**Pull request merged**